app/scripts/models/app.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * App information
```

Comment:
App is super generic, we have an [AppView](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/app.js) which I would assume this model corresponds to. Is this an OAuth app? If so, can you rename it so?